components/hab/src/cli.rs

```diff
@@ -526,6 +526,8 @@ pub fn get(feature_flags: FeatureFlag) -> App<'static, 'static> {
                 (@arg FORCE: --force "Skips checking availability of package and \
                     force uploads, potentially overwriting a stored copy of a package. \
                     (default: false)")
+                (@arg DISABLE_BUILD: --("disable-build")  "Do not create a build group on package upload. This flag has no \
```

Comment:
I think boolean-type flags conform to the `--foo`/`--no-foo` pattern enhances  consistency and usability, making this `--no-build` (which is also a bit shorter).